en/docs/install-and-setup/setup/deployment-best-practices/changing-the-hostname.md

```diff
@@ -74,6 +74,6 @@ To configure hostnames for WSDLs and endpoints, it is recommended to add the fol
 
 ```toml
 [transport.http]
-listener.wsdl_epr_prefix="$ref{server.hostname}"
+listener.wsdl_epr_prefix="http://$ref{server.hostname}"
```

Comment:
Shall we add `https` by default?

Reply:
There is a separate config for https. `listener.secured_wsdl_epr_prefix`. Hence we can use `http://` protocol here.